### PR TITLE
feat: "buy tokens" button

### PR DESCRIPTION
**Pull request by omniwired**
The Osmosis website doesn't have a buy tokens button on mobile.

https://user-images.githubusercontent.com/322761/234165433-cac4bee4-b57e-40fc-9d90-29a8391f7c24.mp4

### CODE REVIEW

File: packages/components/navigation/BuyTokens.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { fontBold16 } from "../../utils/style/fonts";
+import { BrandText } from "../BrandText";
+
+export const BuyTokens: React.FC = () => {
+  return (
+    <TouchableOpacity
+      style={{
+        justifyContent: "center",
+        alignItems: "center",
+        marginBottom: "0.5em",
```
Comment (WaDadidou) [inline packages/components/navigation/BuyTokens.tsx:15]: You should use the same space value as the other eleemnts in the Sidebar.
Like `padding.layout_x1`

File: packages/components/navigation/BuyTokens.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        flexDirection: "row",
+      }}
+      onPress={() => {
+        Linking.openURL("https://frontier.osmosis.zone/?from=OSMO&to=TORI");
+      }}
+    >
+      <CreditCardIcon color={primaryColor} style={{ marginRight: "0.5em" }} />
```
Comment (WaDadidou) [inline packages/components/navigation/BuyTokens.tsx:22]: You should use the same space value as the other elements in the Sidebar.
Like padding.layout_x1

**Comment by omniwired** [comment]
> Could you make this design more "unified", regarding the existing buttons in the Sidebar ? Like :
> 
> * Same font (mainly)
> * Same horizontal align
> * Same icon size
> 
> image

Actually I want it to stand up.
Be different

**Review by WaDadidou — CHANGES_REQUESTED** [review]

File: packages/components/navigation/BuyTokens.tsx (hunk 1 above)
Comment (n0izn0iz) [reply packages/components/navigation/BuyTokens.tsx:15]: furthermost `em` is not supported by native

File: packages/components/navigation/BuyTokens.tsx (hunk 2 above)
Comment (n0izn0iz) [reply packages/components/navigation/BuyTokens.tsx:22]: no `em` see https://reactnative.dev/docs/height-and-width

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

**Comment by omniwired** [comment]
@WaDadidou I tried setting up the font `fontSemibold12` and is too small. IMO

**Pull request merged**